### PR TITLE
walking

### PR DESCRIPTION
**Pull request by RuebenHodges**
walking animations are completely done left and right

### CODE REVIEW

**Comment by RuebenHodges** [comment]
looks good

File: Scene2.js [hunk 1]
```diff
@@ -62,8 +65,13 @@ var Scene2 = new Phaser.Class({
         // Make the camera follow the player
         this.cam.startFollow(this.player);
     
+        //playeranimations
```
Comment (RuebenHodges) [inline Scene2.js:68]: handles all player animations

**Review by RuebenHodges — COMMENTED** [review]
created a file for all player animations, moved something to the global assets loader

**Review by QuintonChinouya — APPROVED** [review]
10/10

**Pull request closed without merging**